src/troute-network/troute/AbstractRouting.py

```diff
@@ -325,8 +314,11 @@ def topobathy_df(self):
                             temp_df.index = new_index
                             cs_id_max = temp_df['cs_id'].max()
                             fill_in_topobathy_df = temp_df[temp_df.cs_id==cs_id_max]
+                            fill_in_topobathy_df.cs_id = fill_in_topobathy_df.cs_id.replace(cs_id_max,1) 
```

Comment:
On this line (317) there is a warning that say: 

> A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead
See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  fill_in_topobathy_df.cs_id = fill_in_topobathy_df.cs_id.replace(cs_id_max,1) 




  to fix this, change line 316 to  ```fill_in_topobathy_df = pd.DataFrame(temp_df[temp_df.cs_id == cs_id_max])```

Reply:
Thanks for this update.